src/main.c

```diff
@@ -0,0 +1,135 @@
+#include <pebble.h>
+#include "main.h"
+
+static Window *s_window;
+static TextLayer *s_label_secondtick, *s_label_animations;
+
+// A struct for our specific settings (see main.h)
+ClaySettings settings;
+
+// Initialize the default settings
+static void prv_default_settings() {
+  settings.BackgroundColor = GColorBlack;
+  settings.ForegroundColor = GColorWhite;
+  settings.SecondTick = false;
+  settings.Animations = false;
+}
+
+// Read settings from persistent storage
+static void prv_load_settings() {
+  // Load the default settings
+  prv_default_settings();
+  // Read settings from persistent storage, if they exist
+  persist_read_data(SETTINGS_KEY, &settings, sizeof(settings));
+}
+
+// Save the settings to persistent storage
+static void prv_save_settings() {
+  persist_write_data(SETTINGS_KEY, &settings, sizeof(settings));
+  // Update the display based on new settings
+  prv_update_display();
+}
+
+// Update the display elements
+static void prv_update_display() {
+  // Background color
+  window_set_background_color(s_window, settings.BackgroundColor);
+  text_layer_set_background_color(s_label_secondtick, settings.BackgroundColor);
```

Comment:
We could set this up as GColorClear at start (which means we don't need to re-set each update)